### ISSUE TITLE
Port test_rawudp to Python3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8545]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8545|
|Type|enhancement|
|Created|2016-07-01 05:57:35Z|
|Branch|https://github.com/twisted/twisted/tree/8545-rodrigc-rawudp|

twisted/pair/test/test_rawudp.py needs to be ported to Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8545 8545
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8545_rodrigc_rawudp 8545-rodrigc-rawudp
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467352655551407 1467352655551407
changetime__1467362125286718 1467362125286718
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/282

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa set owner to @rodrigc|
|-|-|

LGTM, merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 7e1399eb49158bc3786506eff7e313ebb57317a7
```
#!CommitTicketReference repository="" revision="7e1399eb49158bc3786506eff7e313ebb57317a7"
Merge 8545-rodrigc-rawudp: Add twisted.pair.raw and twisted.pair.rawudp to list of modules ported to Python 3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8545
```
